File: Domain/Entities/ET/EvaluateForm.cs
using System.Collections.Generic;

namespace Domain.Entities.ET;

public class EvaluateForm : EntityBase
{
    public string RoleCode { get; set; }
    public string RoleNameTh {  get; set; }
    public string RoleNameEn { get; set; }
    public string LanguageCode { get; set; }
    public bool Active { get; set; }
    public ICollection<EvaluateGroup> EvaluateGroups { get; set; }
}

File: Web/ClientApp/src/app/models/et/evaluateForm.ts
import { EntityBase } from "../entityBase";
import { EvaluateGroup } from "./evaluateGroup";

export class EvaluateForm extends EntityBase {
    roleCode: string;
    roleNameTh: string;
    roleNameEn: string;
    languageCode: string;
    sequeneId: string;
    active: boolean;
    evaluateGroups:  EvaluateGroup[]

    constructor() {
        super();
        this.evaluateGroups = []
    }

}